.CI/create_feedstocks.py

```diff
@@ -253,12 +253,9 @@ def write_token(name, token):
             )
 
     # Update status based on remote
-    subprocess.check_call(['git', 'stash', '--keep-index'])
     subprocess.check_call(['git', 'stash'])
     subprocess.check_call(['git', 'pull', '--rebase'])
-    subprocess.check_call(['git', 'stash', 'pop'])
-    subprocess.check_call(['git', 'add', '.'])
-    subprocess.check_call(['git', 'stash', 'pop'])
+    subprocess.check_call(['git', 'stash', 'pop', '--index'])
```

Comment:
From local testing this seems to combine some of the repeat steps (that also caused the recent failure) into one. This ensures staged changes are still staged, which was our primary concern with PR ( https://github.com/conda-forge/staged-recipes/pull/1472 ). It also restores any unstaged changes as well, but not in the index.